### DISCUSSION TITLE
Grid column style

### DISCUSSION THREAD

**Discussion by curtic00** [vikramlearning/blazorbootstrap#981] [Q&A]
Is this correct ?   
<GridColumn TItem="InvoicesModel" HeaderText="Text"  Style="white-space: pre-line;">
             @context.invText
</GridColumn>

or is there a better way to achive this?

**Comment by gvreddy04** [comment]
@curtic00 Please elaborate on your question with an example or a screenshot.

**Comment by curtic00** [comment]
Hi.  Fixed it by using this format:

                     <GridColumn TItem="InvoicesModel" HeaderText="Text">
                         <span Style="white-space: pre-line;"> @context.invText</span>
                     </GridColumn>

This gave me this:

<img width="977" alt="image" src="https://github.com/user-attachments/assets/ddc449ba-399e-4328-a698-e6a4abe41fd5" />


Thanks for your patience.

BTW: is there a better way to achieve this format?
